modules/help.js

```diff
@@ -68,6 +68,8 @@ class Help {
       if (this._config.get('enableUserManagement') && isPrivileged) {
         response.addField(`\`${prefix}addtime @user TIME_IN_SECONDS\``, 'Add practice time to @user\'s record')
         response.addField(`\`${prefix}deltime @user TIME_IN_SECONDS\``, 'Remove practice time from @user\'s record')
+        response.addField(`\`${prefix}addpoint @user\``, 'Add one literature quiz point to @user\'s record')
+        response.addField(`\`${prefix}delpoint @user\``, 'Remove one literature quiz point from @user\'s record')
```

Comment:
Debatable whether this (and the commands themselves) should be guarded behind `config.get('enableLiteratureQuiz')`. Though I think it likely doesn't matter. WDYT?

Reply:
On second thought, `p!addtime` and `p!deltime` aren't similarly guarded behind the practice module. So it's probably best to stay consistent.